File: refactored/assertion-roulette/CSharpXUnit/CSharpXUnit.Tests/Tests/PollsChoiceModelImpl_ESTest.cs
[Collection("EvoRunner")]
[EvoRunnerParameters(MockJVMNonDeterminism = true, UseVFS = true, UseVNET = true, ResetStaticState = true, SeparateClassLoader = true, UseJEE = true)]
public class PollsChoiceModelImpl_ESTest : PollsChoiceModelImpl_ESTest_scaffolding
{
    [Fact(Timeout = 4000)]
    public void Test00()
    {
        var pollsChoiceImpl0 = new PollsChoiceImpl();
        var pollsChoiceWrapper0 = new PollsChoiceWrapper(pollsChoiceImpl0);
        pollsChoiceWrapper0.SetChoiceId(1L);
        bool boolean0 = pollsChoiceImpl0.Equals(pollsChoiceWrapper0);
        Assert.Equal(1L, pollsChoiceImpl0.GetChoiceId(), "Explanation message");
        Assert.True(boolean0, "Explanation message");
    }
}

File: refactored/assertion-roulette/java-junit/PollsChoiceModelImpl_ESTest.java
@RunWith(EvoRunner.class) @EvoRunnerParameters(mockJVMNonDeterminism = true, useVFS = true, useVNET = true, resetStaticState = true, separateClassLoader = true, useJEE = true) 
public class PollsChoiceModelImpl_ESTest extends PollsChoiceModelImpl_ESTest_scaffolding {
  @Test(timeout = 4000)
  public void test00()  throws Throwable  {
      PollsChoiceImpl pollsChoiceImpl0 = new PollsChoiceImpl();
      PollsChoiceWrapper pollsChoiceWrapper0 = new PollsChoiceWrapper(pollsChoiceImpl0);
      pollsChoiceWrapper0.setChoiceId(1L);
      boolean boolean0 = pollsChoiceImpl0.equals(pollsChoiceWrapper0);
      assertEquals("Explanation message", 1L, pollsChoiceImpl0.getChoiceId());
      assertTrue("Explanation message", boolean0);
  }
}
